stories/core/Table.stories.tsx

```diff
@@ -697,6 +699,7 @@ export const ExpandableSubrows: Story<Partial<TableProps>> = (args) => {
         data={data}
         columns={columns}
         {...args}
+        useVirtualization
```

Comment:
Maybe you can take data generation from `WithPaginator` story and move here to see how virtualization works with big sub-rows list. Or maybe add sub-rows to `Virtualization` sotry.

Reply:
added new story

Reply:
I think it was removed